lib/rack/attack.rb

```diff
@@ -97,7 +97,7 @@ def tracked?(req)
     end
 
     def instrument(req)
-      notifier.instrument('rack.attack', req) if notifier
+      notifier.instrument('rack.attack', :request => req) if notifier
```

Comment:
We should use Ruby 1.9 hash syntax moving forward.
This would become `request: req`